crossword-pdf-uploader/src/main/scala/com/gu/crossword/Lambda.scala

```diff
@@ -36,7 +36,11 @@ class Lambda
     } else if (response.code() == HttpStatus.SC_NOT_FOUND) {
       println(s"Looks like the crossword microapp could not find the relevant crossword for ${crosswordPdfFile.awsKey}. " +
         s"Are you sure its counterpart xml file has been uploaded first?")
-    } else println(s"Upload of crossword PDF location for ${crosswordPdfFile.awsKey} failed.")
+      archiveFailedPdfFiles(crosswordPdfFile.awsKey)
```

Comment:
would suggest not archiving in this case; sometimes the xmls aren't delivered / won't parse and aren't fixed until after publish date. maybe if the date for the pdf has passed by a significant margin (ie. 1week, or 1month) and microapp still doesn't know the corresponding day then archive the failure?

(generally pdfs are much better behaved than xmls, so imo I think there's a less pressing need to archive them)

Reply:
I note that the archive successful pdfs has a check for `isProd` - it might be worth adding the same check onto these archives, just to be triply certain CODE won't start archiving pdfs out of the PROD bucket

Reply:
good points, it seems like the lingering xml files are most of the problem, I added the pdf logic for consistency. Have address these here: https://github.com/guardian/crossword-uploader/pull/31/commits/c7ee41789f86487cb62611f2fe4402e9239093e5